go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/symfony-cli/symfony-cli
 
-go 1.22
+go 1.22.0
```

Comment:
I don't get why this is needed. We've never had an issue with that before, let's revert for now.

Reply:
I just reverted this change, and now I am getting the following error:
```bash
symfony-cli % go test                                       
go: downloading go1.22 (darwin/arm64)
go: download go1.22 for darwin/arm64: toolchain not available
```

I am on macOS 14.4. Should I submit a separate MR for this change?